src/osd/OpRequest.cc

```diff
@@ -38,6 +39,8 @@ OpRequest::OpRequest(Message *req, OpTracker *tracker) :
     reqid = static_cast<MOSDSubOp*>(req)->reqid;
   } else if (req->get_type() == MSG_OSD_REPOP) {
     reqid = static_cast<MOSDRepOp*>(req)->reqid;
+  } else if (req->get_type() == MSG_OSD_REPOPREPLY) {
+    reqid = static_cast<MOSDRepOpReply*>(req)->reqid;
```

Comment:
It would have been a better design if `reqid` was private as it is in `MOSDOp`. Don't know why it is not.